java-checks-test-sources/pom.xml

```diff
@@ -730,6 +730,8 @@
       <groupId>org.mockito</groupId>
       <artifactId>mockito-junit-jupiter</artifactId>
       <version>3.7.7</version>
+      <type>jar</type>
+      <scope>provided</scope>
```

Comment:
For the record, these lines are required in order to have a jar under `java-checks-test-sources\target\test-jars`, jar that will be added in the classpath by the JavaCheckVerifier.